src/coreclr/jit/jitconfigvalues.h

```diff
@@ -454,6 +457,14 @@ CONFIG_STRING(JitNoInlineRange, W("JitNoInlineRange"))
 CONFIG_STRING(JitInlineReplayFile, W("JitInlineReplayFile"))
 #endif // defined(DEBUG) || defined(INLINE_DATA)
 
+CONFIG_INTEGER(JitExtDefaultPolicy, W("JitExtDefaultPolicy"), 1)
+CONFIG_INTEGER(JitExtDefaultPolicyPrejit, W("JitExtDefaultPolicyPrejit"), 0)
```

Comment:
Instead of attaching the policy to prejit flag, would it be better to attach the polity to the existing `CORJIT_FLAG_SPEED_OPT` and `CORJIT_FLAG_SIZE_OPT` options?

I am not fan on attaching the optimization policies to the prejit flag.

Reply:
hm.. for some reason when I invoke crossgen2 with --optimize-time (or --Ot) - CORJIT_FLAG_SPEED_OPT  is still not set in jit 🤔

Reply:
I was wrong, it's set. I verified that -O (blended) and --Os (opt for size) both use DefaultPolicy and --Ot (opt for speed) uses ExtendedDefaultPolicy in crossgen2.